packages/store-engine-sqleet/package.json

```diff
@@ -4,8 +4,7 @@
     "@babel/plugin-proposal-class-properties": "7.5.5",
     "@babel/plugin-proposal-object-rest-spread": "7.5.5",
     "@babel/preset-typescript": "7.3.3",
-    "@types/sql.js": "1.0.2",
-    "sql.js": "https://github.com/pwnsdx/sqleet.js#8075c8ed6a95f6fe7ab7a41703b2fcc3a412be2f",
+    "@wireapp/websql": "https://github.com/wireapp/websql#6425db5537739ff106d37c709ab0e1275d71eb18",
```

Comment:
Why not create a release?

Reply:
I created one now: https://www.npmjs.com/package/@wireapp/websql